.github/workflows/publish-artifacts.yml

```diff
@@ -0,0 +1,173 @@
+name: Publish artifacts to ORAS
+
+on:
+  push:
+    branches:
+    - main
+
+env:
+  RUST_TOOLCHAIN: 1.80.0
+
+jobs:
+  publish-aa:
+    permissions:
+      contents: read
+      packages: write
+      id-token: write
+    strategy:
+      matrix:
+        tee:
+        - none
+        - amd
+        - az-cvm-vtpm
+        - tdx
+        - se
+        - cca
+        include:
+        - tee: none
+          libc: musl
+          arch: x86_64
```

Comment:
Is having a `gnu` `s390x` version of `none` for non-SE s390x possible? I think that combination is the one we use in the peer pods generic s390x podvm image?

Reply:
well, in fact it's not so easy. at the moment we have a 1:1 mapping from tee to arch. now we'd have a none-x64_64 and a none-s390x. we can rename the TEE or are there other options?

Reply:
Yeah, I guess for `TEE_PLATFORM`= none we need to also append the arch and ensure we have the matching logic in kata and CAA where we consume them. We can then maybe revisit this when the `--platform` feature in later oras releases is fully implemented.

Reply:
ok, I ventured a bit into the oci rabbithole. there is a way to get those platform annotations onto the arch images w/ docker, and then we can publish a multi-arch image that can be pulled w/ `oras pull --plaform linux/s390x ...`

we could even misuse the `variant` segment of the platform for TEEs (`linux/s390x/none`) but I'm not sure we want to go there, this should be the variant of the cpu arch I think (arm/v7).